README.md

```diff
@@ -30,8 +30,8 @@ Puppet Strings generates documentation for Puppet code and extensions written in
 
 ### Install Puppet Strings
 
-1. Install the YARD gem by running `gem install yard`
-1. Install the `puppet-strings` gem by running `gem install puppet-strings`
+1. Install the YARD gem by running sudo /opt/puppetlabs/puppet/bin/gem install `yard`
```

Comment:
Probably want the Markdown backtick before sudo:

```
 1. Install the YARD gem by running `sudo /opt/puppetlabs/puppet/bin/gem install yard`
```

> 1. Install the YARD gem by running `sudo /opt/puppetlabs/puppet/bin/gem install yard`

Reply:
good catch @gguillotte, updated.